pyproject.toml

```
@@ -59,14 +59,16 @@ vulture = {version = ">=1.5", optional = true}
 mypy = {version = ">=0.600", optional = true}
 pyright = {version = ">=1.1.3", optional = true}
 pyroma = {version = ">=2.4", optional = true}
+ruff = {version = ">=0.7.1", optional = true}
```

```suggestion
ruff = {version = "*", optional = true}
```

Seems like a limitation that we don't need to impose, but also should not be an issue (as ruff has no dependencies and never will), so maybe using the latest ruff version is better so prospector user have the latest stuff ?